src/App.tsx

```diff
@@ -4,11 +4,20 @@ import { ErrorBoundary } from "react-error-boundary";
 
 import store from "@/stores/redux/store";
 import { errorFallbackRender } from "@/components/error-fallback-render/error-fallback-render";
+import { EmotionCacheProvider } from "@/components/emotion-cache-provider/emotion-cache-provider";
+import { ThemeProvider } from "@emotion/react";
+import { generateTheme } from "@/theme/theme";
 
 export const App = ({ children }: PropsWithChildren): JSX.Element => {
   return (
     <ErrorBoundary fallbackRender={errorFallbackRender}>
-      <Provider store={store}>{children}</Provider>
+      <EmotionCacheProvider>
+        <ThemeProvider theme={generateTheme()}>
```

Comment:
after watching, if you did the configuration correctly press Cmd+S and it will format your code because right now it's not aligned properly

Reply:
resolved